src/martial.py

```diff
@@ -98,9 +109,17 @@ def new_troop(cls, self, nation):
 
         armor = random.choice(nation.armor_list)
 
+        mount = nation.mount_none
+
         elite = random.randint(2, 6)
 
-        return cls(name, strength, health, 0, ranged, speed, discipline, rank_size, ranks, weapons, armor, elite, self.tier + 1, [])
+        if random.randint(0, 100) < (elite * 2) + 5:
+            mount = random.choice(nation.mount_list)
+
+        print str(nation.name) + " created a new unit: " + str(name) + ", armed with: " + str(weapons[0].name) +", " + str(weapons[1].name) + ", "+str(armor.name)+", "+str(mount.name)
```

Comment:
Ditto here as far as making an event and not just printing goes.

Also, as a general rule for printing, please do `print(text)` rather than `print text` so that upgrading to Python 3 is easier if we ever decided to do that.

Reply:
I tried to, but I didn't create it correctly and for the sake did time just made a print.